hazelcast-client-legacy/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -547,14 +547,14 @@ private void handleNamespace(String[] args) {
     private void handleJvm() {
         System.gc();
         println("Memory max: " + Runtime.getRuntime().maxMemory() / ONE_KB / ONE_KB
```

Comment:
I think we can get rid of some newlines here and improve readability with adding `()` around the divisions.